Parsers/ASimRegistryEvent/test/Windows_Event_VimRegistryEvent_SchemaTest.csv

```diff
@@ -0,0 +1,38 @@
+﻿Result
+"(0) Error: Missing mandatory field [ParentProcessId]"
```

Comment:
please check this error

Reply:
done

Reply:
please re-check

Reply:
done

Reply:
Removed this mapping as field is not available in source